src/components/NavTabBar.vue

```diff
@@ -0,0 +1,144 @@
+<script>
+import ExternalLink from '@/components/ExternalLink'
+
+export default {
+  components: {
+    ExternalLink
+  },
+  props: {
+    tabs: {
+      type: Array,
+      required: true
+    },
+    page: {
+      type: String,
+      required: false,
+      default: ''
+    }
+  },
+  data() {
+    return {
+      tab: this.getTab()
+    }
+  },
+  computed: {
+    mainTabs() {
+      return this.tabs.filter(tab => !tab.hidden && tab.align != 'right')
+    },
+    secondaryTabs() {
+      return this.tabs.filter(tab => !tab.hidden && tab.align === 'right')
+    }
+  },
+  watch: {
+    tab(val) {
+      let query = {}
+      if (val) {
+        if (this.$route.params.id) {
+          query = { [val]: this.$route.params.id } // schematic uses this
+        }
+        query = { [val]: '' }
+      }
+      this.$router
+        .replace({
+          query: query
+        })
+        .catch(e => e)
+    },
+    $route() {
+      this.tab = this.getTab()
+    }
+  },
+  methods: {
+    getTab() {
+      if (Object.keys(this.$route.query).length != 0) {
+        return Object.keys(this.$route.query)[0]
+      }
+      return 'overview'
+    }
+  }
+}
+</script>
+
+<template>
+  <v-tabs
+    v-model="tab"
+    class="px-6 mx-auto tabs-border-bottom"
+    :class="$vuetify.breakpoint.smAndDown ? 'tabs-hidden' : ''"
+    style="max-width: 1440px;"
+    light
+  >
+    <v-tabs-slider color="blue"></v-tabs-slider>
+    <v-tab
+      v-for="tb in mainTabs"
+      :key="tb.target"
+      :data-cy="`${page}-${tb.target}-tab`"
+      :href="`#${tb.target}`"
+      :class="$vuetify.breakpoint.smAndDown ? 'tabs-hidden' : ''"
+      :disabled="tb.disabled"
+      :hidden="tb.hidden"
+    >
+      <v-icon left :size="tb.iconSize || 'medium'">{{ tb.icon }}</v-icon>
+      {{ tb.name }}
+      <v-menu
+        open-on-hover
+        :close-on-click="false"
+        :open-on-click="false"
+        :close-on-content-click="false"
+        offset-y
+      >
+        <template #activator="{on}">
+          <div
+            :style="tb.cardText && 'cursor: default; pointer-events: auto;'"
+            v-on="on"
+            @click.stop
+          >
```

Comment:
this feels kinda hacky, but I need to override the `pointer-events:none` of a disabled tab without also making it possible to click on...

Reply:
We've resolved this with some better use of `v-if else` statements on the tabs wrapper.